### ISSUE TITLE
Compilation error with rest package v.1.6.0

### ISSUE THREAD

**Issue by zebox** [go-pkgz/auth#76]
I tried use the `auth` v.1.14.0 in my new project and get compilation error:

```
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\direct.go:124:17: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, *token.User)
	want (http.ResponseWriter, interface {})
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\oauth1.go:163:17: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, *token.User)
	want (http.ResponseWriter, interface {})
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\oauth2.go:214:17: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, *token.User)
	want (http.ResponseWriter, interface {})
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\telegram.go:202:18: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, struct { Token string "json:\"token\""; Bot string "json:\"bot\"" })
	want (http.ResponseWriter, interface {})
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\telegram.go:249:17: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, *token.User)
	want (http.ResponseWriter, interface {})
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\verify.go:130:17: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, *token.User)
	want (http.ResponseWriter, interface {})
***\go\pkg\mod\github.com\go-pkgz\auth@v1.14.0\provider\verify.go:192:17: too many arguments in call to rest.RenderJSON
	have (http.ResponseWriter, *http.Request, rest.JSON)
	want (http.ResponseWriter, interface {})
```

I researched errors and find incompatible dependency in the `auth` package with `rest` package v.1.6.0.
Since this version method **RenderJSON** has two incoming parameters instead three as previously versions.
In the `auth` package dependency for rest package set as v1.5.0

Because I use you `rest` package outside `auth` package, new version `rest` package was fetch to list in `go.mod`
Are you will plan fix this in the near future?

**Comment by zebox** [comment]
fixed typo mistake

**Comment by umputun** [comment]
my bad, missed it. Should have been making v2 but somehow didn't realize this was a breaking change. Anyway, too late for this. Changed dependency to v1.6.0 and adjusted calls.

Pls give it a try (master, not tagged yet) and if all good I'll release it

**Comment by zebox** [comment]
I tried build project again after you made change (direct from master), errors gone!
Thank you very much!

**Comment by umputun** [comment]
released as v1.15.0
